os/binfmt/libelf/libelf_init.c

```diff
@@ -95,6 +95,52 @@
  * Private Functions
  ****************************************************************************/
 
+/****************************************************************************
+ * Name: bin_header_info
+ *
+ * Description:
+ *  Get the binary header info
+ *
+ * Returned Value:
+ *   0 (OK) is returned on success and a negated errno is returned on
+ *   failure.
+ *
+ ****************************************************************************/
+ #ifdef CONFIG_ELF_MTD
```

Comment:
Please define CONFIG_ELF_MTD in Kconfig.

Reply:
Will replace with binary manager config instead.